GOVERNANCE.md

```diff
@@ -1,53 +1,87 @@
-
-## 5. Non-Confidential, Restricted Disclosure.
-
-Information disclosed in connection with any Working Group activity, including but not limited to meetings, Contributions, and submissions, is not confidential, regardless of any markings or statements to the contrary.  Notwithstanding the foregoing, if the Working Group is collaborating via a private repository, the Participants will not make any public disclosures of that information contained in that private repository without the Approval of the Working Group.  
+# FDC3 Governance Policy
+
+This document provides the governance policy for the development of the FDC3 specification and related materials in the FDC3 Standard (the “Working Group”).
+
+## 1.	Roles.
+
+The Working Group includes the following roles:
+
+### 1.1. Participants
+
+“Participants” are those that have made Contributions to the Working Group subject to the [Community Specification Contribution Policy 1.0](https://github.com/finos/standards-project-blueprint/blob/master/governance-documents/6._Contributing.md).  The [FDC3 Standard Working Group](https://github.com/finos/FDC3/issues?q=label%3A%22Standard+WG+Meeting%22) has the specific purpose of defining and releasing subsequent updates to the Standard. In practice, that means people that attend and contribute to meetings, raise issues, pull requests (to submit patches to the Standard) and reviews.
+
+#### How do you become a Participant?
+
+Becoming an FDC3 Participant is as easy as attending a meeting and/or raising issues for changes you'd like see in the Standard, commenting on issues others have raised or even asking questions (which can often result in the clarification of the Standard's documentation to help others with the same questions in future).
+
+### 1.2. Voting Participants
+
+Voting participants are able to vote on changes to the FDC3 Standard (see [Section 2](#2decision-making) below).
+
+- [View the FDC3 Participants GitHub Team](https://github.com/orgs/finos/teams/fdc3-participants)
+
+#### How do you become a Voting Participant?
+
+Participants can become Voting participants by using this link: [fdc3-participants+subscribe@finos.org](mailto:fdc3-participants+subscribe@finos.org?subject=Please%20enroll%20me%20as%20FDC3%20standard%20participant&amp;cc=fdc3-pmc%40finos.org&amp;body=HI%2C%20my%20name%20is%20%3CFirstName%20LastName%3E%20and%20I%27d%20like%20to%20formally%20participate%20to%20the%20FDC3%20standard%20process.%20I%20plan%20to%20contribute%20as%20%3Cindividual%7Con%20behalf%20of%20organizationName%3E%20and%20I%20have%20reviewed%20the%20policies%20described%20at%20https%3A%2F%2Fgithub.com%2Ffinos%2Fcommunity%2Ftree%2Fmaster%2Fgovernance%2FStandards-Projects%20.%20Thank%20you!) to send a templated email email to join the enrolled voting participants group.
```

Comment:
The email link/template here needs to be updated to agree to the CSL license files in this repo rather than the FINOS community one that it links to.

Reply:
So, a link to GOVERNANCE.md instead?